keccak-air/src/air.rs

```diff
@@ -168,5 +179,78 @@ impl<AB: AirBuilder> Air<AB> for KeccakAir {
                 }
             }
         }
```

Comment:
All of the below can go away when we unify the columns as discussed elsewhere in this review.